src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -219,6 +219,11 @@ void FullyConnected::getSupportedDescriptors() {
         if (one_of(outputDataType , memory::data_type::u8, memory::data_type::s8)) {
             outputDataType = memory::data_type::bf16;
         }
+        // W.A. WeightsDecompression not supported on avx2_vnni_2
```

Comment:
1. In fact this is not WA, but valid precision adjustment
2. Please create feature request and add ticket number here.
3. I would propose to change the condition: `useWeightsDecompressionImpl && !dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16)` as long as corresponding impls are enabled only for that ISA
https://github.com/openvinotoolkit/oneDNN/blob/cb3060bbf4694e46a1359a3d4dfe70500818f72d/src/cpu/cpu_inner_product_list.cpp#L93

Reply:
have updated, and linked this previously created ticket to Epic CVS-55480